src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -186,15 +186,18 @@ export class WizardModal extends Modal {
 		if (validate && !await this.validateNavigation(index)) {
 			return;
 		}
+
+		let dialogPaneToShow: DialogPane | undefined = undefined;
 		this._dialogPanes.forEach((dialogPane, page) => {
 			if (page === pageToShow) {
+				dialogPaneToShow = dialogPane;
 				dialogPane.show(focus);
 			} else {
 				dialogPane.hide();
 			}
 		});
 
-		status(localize('wizardPageChangedMessage', "Step {0}: {1}", index + 1, pageToShow.title));
+		status(localize('wizardPageChangedMessage', "{0} {1}", dialogPaneToShow.pageNumberDisplayText, dialogPaneToShow.title));
```

Comment:
I guess for overall safety probably worth it to not do this if dialogPaneToShow is undefined. (would never expect that to actually happen but theoretically it could...)